NEWS

```diff
@@ -1,2 +1,2 @@
-Changes in version 0.99.0 (2020-07-02)
```

Comment:
Make a new line instead of changing the past change log

Reply:
See 5760d0e